### PR TITLE
Tehtris: add documentation

### PR DESCRIPTION
**Pull request by squioc**
Add the documentation to configure the collection of Tehtris EDR events

### CODE REVIEW

**Review by gbossert — CHANGES_REQUESTED** [review]
I don't see the Tehtris module doc

File: docs/xdr/features/collect/integrations/endpoint/tehtris_edr.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+uuid: 8d8f040d-6a75-4bf4-bf1d-772e9a30f0dd
+name: Tehtris Endpoint Detection & Reponse
+type: intake
+
+## Overview
+
+Tehtris EDR is a security product to monitor, detects and mitigates threats on endpoints. It reduces the attack surface and prevent attacks.
+Combined with Tehtris EPP, it detects malwares and prevent exploits on your devices.
```
[the remark this answers is not in the document]
Comment (squioc) [reply docs/xdr/features/collect/integrations/endpoint/tehtris_edr.md:8]: Applied

File: docs/xdr/features/collect/integrations/endpoint/tehtris_edr.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+uuid: 8d8f040d-6a75-4bf4-bf1d-772e9a30f0dd
+name: Tehtris Endpoint Detection & Reponse
+type: intake
+
+## Overview
+
+Tehtris EDR is a security product to monitor, detects and mitigates threats on endpoints. It reduces the attack surface and prevent attacks.
```
[the remark this answers is not in the document]
Comment (squioc) [reply docs/xdr/features/collect/integrations/endpoint/tehtris_edr.md:7]: Applied

**Review by PierrickV — APPROVED** [review]
nice work!

**Review by gbossert — APPROVED** [review]

**Pull request merged**